apps/frontend/src/libs/components/balance-wheel-chart/libs/helpers/generate-gradient-color/label-to-background-color.map.ts

```diff
@@ -0,0 +1,34 @@
+import { FALLBACK_BACKGROUND_COLOR } from "~/libs/components/balance-wheel-chart/libs/constants/constants.js";
+import { ChartSliceColor } from "~/libs/components/balance-wheel-chart/libs/enums/enums.js";
+
+const labelToBackgroundColor = (
```

Comment:
```suggestion
const getBackgroundColorFromLabel = (
```
Change the file name to `get-background-color-from-label.helper.ts`
Move it outside of `generate-gradient-color` folder and create a new `get-background-color-from-label` folder for it